content/communities/open-gov.md

```diff
@@ -21,18 +21,6 @@ topics:
 # 1 -- visible
 weight: 1
 
```

Comment:
This community also seems to be missing a blurb.

Reply:
See first comment.